controllers/server-controllers.js

```diff
@@ -59,6 +59,22 @@ exports.getArticleById= (req, res, next) => {
     .catch(next)
 }
 
+exports.patchArticleById= (req, res, next) => {
+    const newVote=req.body
+    const {article_id}=req.params
+    const commentPromise=[updateArticleById(newVote,article_id)]
+    if (article_id)
+    {
+        commentPromise.push(checkArticle_idExists(article_id))
+    }
```

Comment:
Do you need this if statement? Will there even not be an article id, given that the route is `/api/articles/:article_id`??